server/database/queries/offers/saved-offers.js

```diff
@@ -0,0 +1,12 @@
+const dbconnection = require('../../config/db_connection');
+
+const addSavedOffers = (savedOfferInfo) => {
+  const queryValues = Object.values(savedOfferInfo);
+  const sql = {
+    text: ' INSERT INTO saved_offer(member_id, offet_id) VALUES ($1, $2) RETURNING *',
```

Comment:
@Fatmasiam I have one more idea to make this safer: if / when authentication is built, use the authenticated member_id from the token instead of having the client pass member_id in the request body.  That way, user A can't manually POST and change user B's saved offers :)